src/integration/prepareApiReferences.js

```diff
@@ -73,23 +73,29 @@ function ramlToHtml(files, name){
   const config = raml2html.getDefaultConfig(path.resolve(__dirname, '../raml2htmlTemplates/template.nunjucks'), path.resolve(__dirname, '../raml2htmlTemplates'));
 
   return (cb) => {
-    validator.dirCheck(files, (err) => {
+    const ramlFilePath = `${files}/api.raml`;
+
+    validator.fileCheck(ramlFilePath, (err) => {
 
       if (err) return cb(null, name);
 
-      raml2html.render(`${files}/api.raml`, config)
+      raml2html.render(ramlFilePath, config)
         .then((result) => {
           fs.writeFile(`${files}/apireferenceTempContent.html`, result, 'utf-8', cb);
-        }, cb)
-        .catch(cb);
+        }, (err) => {
+
```

Comment:
It would be nice to log the information:
'The APIReference section will not be displayed for {nameOfTheService}'